### ISSUE TITLE
Using earlier versons of action always uses latest version of container

### ISSUE THREAD

**Issue by gardar** [gofrolist/molecule-action#140]
When attempting to use older versions of the action, (either to get a version that has older versions of ansible/molecule or to pin the workflow to a known working version of the action), it always results in the latest version of the container.

Being able to pin a specific verios of the action/container would avoid unexpected issues such as: #138 

It would be good if the versions of the action would either match the corresponding container version or if it would be possible to overwrite the container version used via a env variable for example.

Perhaps something like this could work?

```yaml
runs:
  using: docker
  image: docker://gofrolist/molecule:v${MOLECULE_CONTAINER_VERSION}
  env:
    ANSIBLE_ASYNC_DIR: "/tmp/.ansible_async"
    ANSIBLE_FORCE_COLOR: "1"
    MOLECULE_CONTAINER_VERSION: "2"
```

**Comment by disaster37** [comment]
I agree, it can be usefull to have ability to set our custom docker image that embed the exactly molecule, ansible version we use. It can also permit to use specific molecule plugin like molecule-goss, etc .

**Comment by davestephens** [comment]
Another vote for this. Spent the last hour scratching my head as to why my attempts to use a previous version of the action were still pulling the latest Molecule. Seems a bit pointless having any versioning at all if the Docker image in use is pinned to the latest v2.

**Comment by gofrolist** [comment]
I tried your suggestion, but unfortunately, GitHub Actions does not support variables in the `image` field.
So, instead of using a variable, I created a template for `action.yml` and replace a Docker tag during the release.
This change has been implemented in PR #308.
Starting from version `v2.7.1`, this molecule-action will point to the corresponding Docker image.
